python/jetToolbox_cff.py

```diff
@@ -348,22 +349,28 @@ def jetToolbox( proc, jetType, jetSequence, outputFile,
 		patJets = 'updatedPatJets'
 
 		if updateCollectionSubjets:
-			print '|---- jetToolBox: JETTOOLBOX IS UPDATING '+updateCollection+' collection for subjets/groomers.'
+			print '|---- jetToolBox: JETTOOLBOX IS UPDATING '+updateCollectionSubjets+' collection for subjets/groomers.'
+			if 'SoftDrop' in updateCollectionSubjets: updateSubjetLabel = 'SoftDrop'
+			else: updateSubjetLabel = 'Pruned'
 			updateJetCollection(
 					proc,
-					jetSource = cms.InputTag( updateCollectionSubjets, 'SubJets' ),
-					labelName = jetALGO+'PF'+PUMethod+'SoftDropPacked',
+					jetSource = cms.InputTag( updateCollectionSubjets ),
+					labelName = jetALGO+'PF'+PUMethod+updateSubjetLabel+'Packed',
 					jetCorrections = JEC, 
-		#			btagDiscriminators = bTagDiscriminators,
+					explicitJTA = True,
+					fatJets = cms.InputTag( updateCollection ),
+					rParam = subjetSize,
+					algo = jetALGO,
```

Comment:
again, I think this subjetSize is not need it at all. I will remove it before merging.